src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -9232,3 +9235,92 @@ definitions:
       id:
         type: string
         description: The unique identifier of the action pending execution
+  Definition:
+    required:
+      - typeId
+    properties:
+      typeId:
+        description: the id of the type
+        type: string
+      title:
+        description: the human-readable class title
+        type: string
+      parentName:
+        description: name of the parent class
+        type: string
+      description:
+        description: description of the class
+        type: string
+      archive:
+        description: define if the type is archived on delete
+        type: boolean
+      properties:
+        description: list of properties referring to the type hierarchy
+        type: array
+        items:
+          $ref: '#/definitions/Property'
+  Property:
+    type: object
+    required:
+      - id
+    properties:
+      id:
+        type: string
+      title:
+        description: the human-readable title
+        type: string
+      description:
+        description: the human-readable description
+        type: string
+      defaultValue:
+        description:  the default value
+        type: string
+      dataType:
+        description: the name of the property type (i.g. d:text)
+        type: string
+      class:
+        description:  the class of the property
+        type: string
+      override:
```

Comment:
I know it's not 100% consistent across the API but we try and prefix boolean properties with "is", see #/definitions/Node. As this is part of the node API we should be consistent.